src/isomorphic/React.js

```diff
@@ -28,6 +28,13 @@ var createFactory = ReactElement.createFactory;
 var cloneElement = ReactElement.cloneElement;
 
 if (__DEV__) {
+  console.log(
```

Comment:
We should use `warning` to remain consistent with the rest of the codebase. Since this is unconditional you would just do:

```js
warning(false, "message")
```

But we'd likely want to dedupe this error message if accepted. See how [the `React.__spread` warning](https://github.com/rauchg/react/blob/8d59cc29218526de247c09ce1902f6bef8bb724f/src/isomorphic/React.js#L46-L59) handles it below.